jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -248,7 +254,7 @@ def test_load_spectrum_list_directory(tmpdir, specviz_helper):
 
     with pytest.warns(UserWarning, match='SRCTYPE is missing or UNKNOWN in JWST x1d loader'):
         specviz_helper.load_spectrum(data_path)
-    assert len(specviz_helper.app.data_collection) == 3
+    assert len(specviz_helper.app.data_collection) == 41
```

Comment:
According to https://github.com/spacetelescope/jdaviz/pull/1659#issuecomment-1284504407 , no combined anything should be produced here. 🤷 

```suggestion
    assert len(specviz_helper.app.data_collection) == 39
```

Reply:
In the latest commits I've changed this behavior (again). See below!